lib/target/generic_cpp/generic_cxx_codegen.cc

```diff
@@ -532,12 +532,18 @@ static void EmitComputationItems(std::ostream* os, const CXXCodeGenOpts& opts) {
     *os << "bool use_ipu_model = " << opts.use_ipu_model << ";\n";
     *os << "int ipu_num = " << opts.ipu_num << ";\n";
     *os << "int batches_per_step = " << opts.batches_per_step << ";\n";
+    *os << "bool enable_engine_cache = " << opts.enable_engine_cache << ";\n";
+    *os << "const char* cache_dir = " << opts.cache_dir << ";\n";
```

Comment:
HALO won't generate such code in future. These will be done by ODLA runtime APIs

Reply:
ok